packages/axelar-wasm-std/src/address.rs

```diff
@@ -29,6 +31,13 @@ pub fn validate_address(address: &str, format: &AddressFormat) -> Result<(), Err
             SuiAddress::from_str(address)
                 .change_context(Error::InvalidAddress(address.to_string()))?;
         }
+        AddressFormat::Stellar => {
+            if address != address.to_uppercase() {
+                bail!(Error::InvalidAddress(address.to_string()))
+            }
+            ScAddress::from_str(address)
```

Comment:
did you look into this implementation for any malleability concerns for sanity?

Reply:
yes, only thing I come up is non-canonical string representation, it's case insensitive. didn't find anything else.